### PR TITLE
LF-22851: Adding password error strings

### PR DESCRIPTION
**Pull request by markdoten**
https://livefyre.atlassian.net/browse/LF-22851

### CODE REVIEW

**Comment by kwalter37** [comment]
Good to merge when you are ready.

File: nls/identity.json [hunk 1]
```diff
@@ -80,6 +82,19 @@
     "therewasanerror": {"key": "therewasanerror", "default": "There was an error", "description": ""},
     "thisfieldisrequired": {"key": "thisfieldisrequired", "default": "This field is required.", "description": ""},
     "validfileextensions": {"key": "validfileextensions", "default": "Valid file extensions", "description": ""},
-    "valuemustmatch": {"key": "valuemustmatch", "default": "Value must match", "description": ""}
+    "valuemustmatch": {"key": "valuemustmatch", "default": "Value must match", "description": ""},
+    "passwordLength": {"key": "passwordLength", "default": "be 6 to 32 characters long.", "description": "Password required length error message shown in password popover"},
```
Comment (kwalter37) [inline nls/identity.json]: How does that look on the screen? bulleted list?

File: nls/identity.json [hunk 2]
```diff
@@ -80,6 +82,19 @@
...
+    "passwordErrorContainsLastName": {"key": "passwordErrorContainsLastName", "default": "The password you entered contains either your username, first name or last name. For security reasons, please enter a password that does not contain your username, first name or last name.<br><br>Please also remember that your password needs to contain:<br>6 to 32 characters<br>An uppercase character<br>A lowercase character<br>A symbol", "description": "Error message shown on the reset password flow if the user's password contains their last name"},
+    "passwordErrorContainsUsername": {"key": "passwordErrorContainsUsername", "default": "The password you entered contains either your username, first name or last name. For security reasons, please enter a password that does not contain your username, first name or last name.<br><br>Please also remember that your password needs to contain:<br>6 to 32 characters<br>An uppercase character<br>A lowercase character<br>A symbol", "description": "Error message shown on the reset password flow if the user's password contains their username"},
```
Comment (kwalter37) [inline nls/identity.json]: The 3 strings above the exact same on purpose?

File: nls/identity.json (hunk 1 above)
Comment (markdoten) [inline nls/identity.json]: Yes, these ones are part of a bulleted list

File: nls/identity.json (hunk 2 above)
Comment (markdoten) [inline nls/identity.json]: Yes, I decided that I wanted to provide more customizability for users. First name, last name, and username are currently all exactly the same based on what Liana wanted to do. That doesn't mean that users shouldn't be able to make them different.

**Pull request merged**